packages/twenty-front/src/modules/settings/data-model/constants/SettingsFieldCurrencyCodes.ts

```diff
@@ -91,6 +129,143 @@ export const SETTINGS_FIELD_CURRENCY_CODES: Record<
   },
   AUD: {
     label: 'Australian dollar',
-    Icon: IconCurrencyDollar,
+    Icon: IconCurrencyDollarAustralian,
+  },
+  DZD: {
+    label: 'Algerian Dinar',
+    Icon: IconCoins,
   },
+  PLN: { label: 'Polish złoty', Icon: IconCurrencyZloty },
```

Comment:
I like the 1 liner format, could we wlso format the others + order them by alphabetical order to ease the maintenance?

Reply:
Sure, oneliner is a fast fix, about order, I left them as they are because after all, that's the order shown for the end users, I didn't want to change first few currencies as they're most popular, if you think it's okay to sort them alphabetically and thus, change the order for end users, then I can do it.

Reply:
I think we should be consistent and order them by alphabetical order, it's fine to change the order right now

Reply:
I'm on it